pkg/client/testdata/custom-systemd-logs-image.golden

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
All these were just artifacts of how the string was defined before;  now there isn't a need for the random empty line at the start.